api_compatibility_tests/setup.py

```diff
@@ -53,11 +53,14 @@ def get_requirements():
         "Programming Language :: Python :: 3.5",
```

Comment:
we should probably remove this line?

Reply:
This is for the API Compatibility module. We never actually upload this to PyPi, so this part should not matter.